### PR TITLE
Added Gerrit support for both US&DS

### PR DESCRIPTION
**Pull request by Omeramsc**
fix CPORG-9

### CODE REVIEW

File: base.py [hunk 1]
```diff
@@ -4,19 +4,21 @@
...
-        self.site_url = site_url
+        self.host = site_url
 
     @abstractmethod
     def _connect(self):
         pass
 
     @abstractmethod
-    def get_cc_from_org(self):
+    def get_cc(self):
```
Comment (lioramilbaum) [inline base.py]: The function returns multiple ccs. Update its name.

File: main.py [hunk 2]
```diff
@@ -32,19 +34,34 @@ def main():
     gh = base.GitHub()
     gh.set_orgs(GITHUB_ORGS)
 
-    if gl.groups or gh.orgs:
+    ger_ds = base.Gerrit('gerrit.ovirt.org')
```
Comment (lioramilbaum) [inline main.py]: this is the us gerrit host

File: main.py [hunk 3]
```diff
@@ -32,19 +34,34 @@ def main():
...
+    ger_ds.set_orgs(GERRIT_UPSTREAM_PROJECTS)
```
Comment (lioramilbaum) [inline main.py]: again there is a mix between ds and us

File: main.py (hunk 2 above)
Comment (Omeramsc) [reply main.py]: fixed

File: main.py (hunk 3 above)
Comment (Omeramsc) [reply main.py]: fixed

**Comment by lioramilbaum** [comment]
Can you please add me to the org? I don't have permissions to resolve comments nor to merging the PR.

**Comment by Omeramsc** [comment]
> Can you please add me to the org? I don't have permissions to resolve comments nor to merging the PR.

Weird, was sure I did it before.
anyway, done.

**Pull request merged**